package.json

```diff
@@ -104,6 +104,7 @@
   "dependencies": {
     "base-64": "^1.0.0",
     "events": "^3.3.0",
+    "exponential-backoff": "^3.1.0",
```

Comment:
Why does this PR add this dependency? I don't see it used

Reply:
I think that's used in Valters pr that this branch was based on. As we intend to have exponential backoff with reconnects. If you don't mind I would just keep it here.